package.json

```diff
@@ -36,6 +37,7 @@
     "@types/typescript": "^2.0.0",
     "babel-loader": "^8.2.2",
     "babel-plugin-module-resolver": "^4.1.0",
+    "charcodes": "^0.2.0",
```

Comment:
Why was this added?

Reply:
That is a dependency used in one of the babel packages.

Reply:
Running `yarn install` in the `babel` directory should be enough.  That should probably be added to the `prepare` script you added instead adding individual babel dependencies here.

Reply:
Done
https://github.com/Khan/flow-to-ts/pull/271/commits/23ced32d7359837994fef21610518dcf6d4722b1